doi.jq

```diff
@@ -45,6 +41,12 @@ def repos_anti_subset:
 		"rakudo-star",
 		"silverpeas",
 
+		# deprecated
+		"adminer",
+		"centos", # direct children: eclipse-temurin ibm-semeru-runtimes percona swift
+		"php-zendserver",
+		"sl",
```

Comment:
The sl bashbrew file has already been removed from docker-library/official-images/library.

Reply:
Im more grouping them at the bottom until they can safely be removed